src/coreclr/src/vm/dllimport.cpp

```diff
@@ -6328,6 +6331,19 @@ namespace
         }
 #endif
 
+        if (g_hostpolicy_embedded)
+        {
+#if defined(TARGET_LINUX)
```

Comment:
Would it be a good idea to invert this condition to make this logic future proof; such that all OS except Windows get `libhostpolicy`?

Reply:
Yes, I think everything but Windows should use `libhostpolicy`

Reply:
Is this actually the name used in this context? - not `libhostpolicy.so` or something with a path?

In the case above we could, in theory, see different file names, but we intentionally only recognize the form used in `Interop.Libraries.cs`.
Jut wonder whether there is anything to worry about hostpolicy naming.

Reply:
The story is the same - we don't expect any other code but corelib to call into hostpolicy directly, so these strings need to match what's in `Interop.Libraries`, but nothing else.

I fixed the condition to be for Windows - should also fix the build failures in CI